README.md

```diff
@@ -2,17 +2,8 @@ This is not a full portage configuration. It contains only those parts that are
 necessary to set up a gentoo github mirror based sync system:
 
 * Configures Portage to sync via https://github.com/gentoo/gentoo.git URL.
-* Updates metadata-cache (the first update might take long, all subsequent ones not).
+* Updates metadata-cache.
 * Updates the dtd directory.
 * Updates the glsa directory.
 * Updates `herds.xml` file.
 * Updates the news directory.
-
-## Notes ##
```

Comment:
But the `sync_gentoo_cache` script would need several adjustments to work with overlays in general. Previously, that logic was embedded, see https://github.com/hasufell/portage-gentoo-git-config/blob/d5ec4ddb985c79e663e5e3621348a12790f7245a/etc/portage/postsync.d/sync_cache but it's not that generic anymore, since people thought it doesn't belong there.

For an overlay version, we would want:
```
#!/bin/bash

repository_name="${1}"

source /lib/gentoo/functions.sh

# Number of jobs for egencache, default is number or processors.
parallel_jobs="$(nproc)"

ebegin "Updating metadata cache for ${repository_name}"
egencache --jobs="${parallel_jobs}" --repo="${repository_name}" --update --update-use-local-desc
eend $?
```